CHANGELOG.md

```diff
@@ -4,9 +4,27 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+## Unreleased
+
+## [v1.4.0] - 2023-08-17
+
+### Notable Changes
+
+This version of the CMA java client works with cumulus-message-adapter
+[v2.0.2](https://github.com/nasa/cumulus-message-adapter/releases/tag/v2.0.2) or later,
```

Comment:
just making sure this is the right version, because we were talking about the cma update to v2.0.3

Reply:
Updated the changelog.  Java 8 Amazon Linux 1 (PODAAC uses currently) won't work with CMA v2.0.3 due to the python incompatibility.